src/NzbDrone.Core/Datastore/Migration/156_update_download_client_ssl_options.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System.Collections.Generic;
+using System.Data;
+using FluentMigrator;
+using Newtonsoft.Json.Linq;
+using NzbDrone.Common.Extensions;
+using NzbDrone.Common.Serializer;
+using NzbDrone.Core.Datastore.Migration.Framework;
+
+namespace NzbDrone.Core.Datastore.Migration
+{
+    [Migration(156)]
+    public class update_download_client_ssl_options : NzbDroneMigrationBase
```

Comment:
I don't think this migration is necessary, it should inherit the default from settings when parsed from the DB.

Reply:
removed